runtime/nls/j9vm/j9vm.nls

```diff
@@ -1654,6 +1654,20 @@ J9NLS_VM_IDLE_TUNING_PLATFORM_OR_GC_POLICY_NOT_SUPPORTED.system_action=The JVM w
 J9NLS_VM_IDLE_TUNING_PLATFORM_OR_GC_POLICY_NOT_SUPPORTED.user_response=Refer to the documentation for supported platforms and GC policies.
 # END NON-TRANSLATABLE
 
+J9NLS_VM_OPTIONS_MUST_BE_NUMBER=%s must be followed by a number
+# START NON-TRANSLATABLE
+J9NLS_VM_OPTIONS_MUST_BE_NUMBER.explanation=The specified option must be immediately followed by a number
+J9NLS_VM_OPTIONS_MUST_BE_NUMBER.system_action=The JVM will terminate
+J9NLS_VM_OPTIONS_MUST_BE_NUMBER.user_response=Correct the option
+# END NON-TRANSLATABLE
+
+J9NLS_VM_OPTIONS_INCORRECT_MEMORY_SIZE=Value for '%s' is not correct
+# START NON-TRANSLATABLE
+J9NLS_VM_OPTIONS_INCORRECT_MEMORY_SIZE.explanation=Value specified for the option is not correct
+J9NLS_VM_OPTIONS_INCORRECT_MEMORY_SIZE.system_action=The JVM will terminate
+J9NLS_VM_OPTIONS_INCORRECT_MEMORY_SIZE.user_response=Correct the option
+# END NON-TRANSLATABLE
+
```

Comment:
Are this NLS messages used somewhere? I can not find any

Reply:
Thanks. See https://github.com/eclipse/openj9/pull/7476/commits/cad6beaec1cc182b1a5325f005ef62dba56d2810

Reply:
Removed. I believe it was remnants of an earlier attempt to implement error processing. 
https://github.com/eclipse/openj9/pull/7476/commits/f7d88ae9047fd371fb725b18ab7c63187e7765db